src/pages/Reservation/components/Button.js

```diff
@@ -13,4 +13,7 @@ const StyledButton = styled.input`
   padding: 10px 12px;
   font-size: 16px;
   font-weight: bold;
+  color: ${props => (props.color ? props.color : 'black')};
+  background-color: ${props => (props.bgc ? props.bgc : 'white')};
```

Comment:
```suggestion
  background-color: ${props => (props.backgroundColor ? props.backgroundColor : 'white')};
```
bgc 보다는 다른 사람이 봐도 명확할 수 있도록 backgroundColor라고 써주는게 좋겠습니다